lib/rspec/rails/fixture_support.rb

```diff
@@ -13,7 +13,11 @@ module FixtureSupport
         # Monkey patched to avoid collisions with 'let(:name)' in Rails 6.1 and after
         # and let(:method_name) before Rails 6.1.
         def run_in_transaction?
-          use_transactional_tests && !self.class.uses_transaction?(self)
+          if ::Rails::VERSION::STRING > '5'
```

Comment:
Generally, we define conditional methods rather than use run time checks, and this should probably check for the existence of `use_transactional_tests` directly rather than a version, although if you leave it on a version it should be `>= 5`